action-delay-job-completion/action.yaml

```diff
@@ -22,9 +22,6 @@ runs:
         EOF
         )
         echo "::notice title=Delay workflow completion::${MESSAGE}"
-        Delete the file '$GITHUB_WORKSPACE/timeout_date' to cancel.
```

Comment:
Shouldn't you keep this enclosed with `echo ""`?

Reply:
This in already in `${MESSAGE}`